src/app/(auth)/signup/page.tsx

```diff
@@ -29,49 +31,83 @@ export default function SignUp() {
     if (error) {
       throw new Error(`An error occurred trying to sign up: ${error.message}`);
     }
-    push('/');
+    setEmailSentCount(1);
```

Comment:
Alternative to getting rid of `push` -- we could use it to push a link with a search query to the browser history (like `/signup?sent=true`) to update the state. I think it'll be more user-friendly as they can use the back-link to navigate back to the sign-up page. Though, that could potentially bring in more room for bugs depending on how it's handled. This is definitely out of scope and niche, but just wanna flag it as a potential stretch feature!

Reply:
logged on [here](https://www.notion.so/calblueprint/STRETCH-70180e69b2914fada3f987e75295c7d9) - comprehensive list of stretch features that are not on the flows